pkg/model/components/kubecontrollermanager_test.go

```diff
@@ -59,6 +59,42 @@ func Test_Build_KCM_Builder_Lower_Version(t *testing.T) {
 		t.Fatalf("AttachDetachReconcileSyncPeriod should not be set for old kubernetes version %s", spec.KubernetesVersion)
 	}
 
+	c = buildCluster()
+
+	kcm = &KubeControllerManagerOptionsBuilder{
+		Context: &OptionsContext{
+			Cluster: c,
+		},
+	}
+
+	spec = c.Spec
+
+	spec.KubernetesVersion = "v1.5.0"
```

Comment:
I think you can replace this with a loop

`for _, k := range []string{ "v.1.5.0", "v1.5.1" }`